module/documents/actor.mjs

```diff
@@ -111,18 +114,68 @@ export default class ActorEd extends Actor {
     const attributeStep = this.system.attributes[ability.system.attribute].step;
     const abilityFinalStep = attributeStep + ability.system.level;
     const difficulty = await ability.system.getDifficulty();
+    let targets = game.users.current.targets
+    let chatFlavor = "";
+    if ( !ability.system.combatAbilityType ) {
```

Comment:
This is too big and distracts from the actual task of this function. This should be factored out. And since the task is needed in `rollEquipment` as well, it should be a generalized function that can return chatFlavor for a number of different parameters.

Reply:
I'm with you, I took it out and put it into #758